src/marketing/price-rules-cart.md

```diff
@@ -7,16 +7,16 @@ Cart price rules apply discounts to items in the shopping cart, based on a set o
 ![Example storefront - cart apply coupon]({% link images/images/storefront-cart-apply-coupon.png %}){: .zoom}
 _Apply Coupon in Cart_
 
-<!--{% if "Default.CE Only" contains site.edition %}-->
-![Cart price rule]({% link images/images/price-rule-cart.png %}){: .zoom}
-_Cart Price Rules_
-<!--{% endif %}-->
-<!--{% if "Default.EE-B2B" contains site.edition %}-->
-![Cart price rule]({% link images/images/price-rules-cart-ee.png %}){: .zoom}
-_Cart Price Rules_
-<!--{% endif %}-->
+## To access Cart Price Rules:
```

Comment:
```suggestion
## Access Cart Price Rules
```

Reply:
All of the instructions begin with"To do this or that."   It's not a heading. Am I using the wrong style?